dev/PushNotifications/PushNotificationsLongRunningTask/NotificationListener.cpp

```diff
@@ -50,3 +44,15 @@ STDMETHODIMP_(HRESULT __stdcall) NotificationListener::OnRawNotificationReceived
     return S_OK;
 }
 CATCH_RETURN()
+
+const std::string NotificationListener::ConvertProcessNameToUtf8String()
```

Comment:
Not needed. Just use the helper method provided by @sharath2727 in his PR.

Reply:
I would remove this method and just call his Helper with m_processName wherever needed

Reply:
> use the helper method provided by @sharath2727 Sharath Manchala FTE in his PR.

link?

Reply:
This function was used before the Protocol activation fixes. @pmpurifoy needs to fix the merge with WNP_LRP (i.e. remove this function).